### PR TITLE
hotfix/Fix_SDL_core_develop

### PR DESCRIPTION
**Pull request by OHerasym**
Added changes to application_manager/test/CMakeList.txt

Related:
        APPLINK-20010
Reason:
             Add code to create directories for creating symlinks.
             Fix develop build with tests.

### CODE REVIEW

**Comment by OHerasym** [comment]
Please review: @LuxoftAKutsan , @anosach-luxoft , @AByzhynar

**Comment by AByzhynar** [comment]
Please add to review @Kozoriz and  @malirod

File: src/components/application_manager/test/CMakeLists.txt [hunk 1]
```diff
@@ -31,6 +31,23 @@
 # TODO{ALeshin}: APPLINK-10792. Do not write tests which use
 # application manager(AM) singleton while refactoring of AM is finished.
```
Comment (AByzhynar) [inline src/components/application_manager/test/CMakeLists.txt]: Change year to 2015 at least. May be 2016? )

**Comment by OHerasym** [comment]
Please review: @malirod, @Kozoriz, @VVeremjova

**Comment by Kozoriz** [comment]
reviewed

**Comment by OHerasym** [comment]
@AByzhynar , 9ddb733 , Changed to 2016

**Comment by LuxoftAKutsan** [comment]
reviewed

**Comment by OHerasym** [comment]
@malirod, 9ddb733, deleted unused include from CMakeList.txt

**Comment by malirod** [comment]
Reviewed

**Comment by VVeremjova** [comment]
Reviewed

**Comment by AByzhynar** [comment]
Reviewed

**Comment by anosach-luxoft** [comment]
@OHerasym, please extend pull request description with the reason of adding this changes

**Comment by OHerasym** [comment]
@anosach-luxoft , d6e3876 , Done

**Comment by OHerasym** [comment]
@anosach-luxoft , d6e3876 , Done

**Pull request merged**